### DISCUSSION TITLE
Can't install akka.net  project with target framework 4.7.2

### DISCUSSION THREAD

**Discussion by qthanpt** [akkadotnet/akka.net#7189] [Q&A] [answered]
I create a console project with .net framework 4.7.2 then install akka.net from nuget.org but receive error on output window
`
Attempting to gather dependency information for package 'Akka.1.5.20' with respect to project 'SimEngine', targeting '.NETFramework,Version=v4.7.2'
Gathering dependency information took 620 ms
Attempting to resolve dependencies for package 'Akka.1.5.20' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'Akka.1.5.20'
Resolved actions to install package 'Akka.1.5.20'
  GET https://www.nuget.org/api/v2/package/Akka/1.5.20
Install failed. Rolling back...
Package 'Akka.Analyzers.0.2.4' does not exist in project 'SimEngine'
Package 'Akka.Analyzers.0.2.4' does not exist in folder 'D:\projects\cscada\git\tools\MxAccessSimulator\packages'
Executing nuget actions took 271 ms
Could not install package 'Akka.Analyzers 0.2.4'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.7.2', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
Time Elapsed: 00:00:01.0275497
========== Finished ==========
`

**Comment by Arkatufus** [comment]
Akka.NET should not have any problem with .NET 4.7.2, can you check your NuGet settings?

**Reply by Aaronontheweb** [reply]
> 'Akka.Analyzers 0.2.4'.

@Arkatufus don't we have tests now to ensure that Akka.Analyzers works properly on all .NET and Roslyn versions that .NET Standard 2.0 supports? I recall we had to add those last quarter.

**Reply by qthanpt** [reply]
Can you provide correct nuget settings?

**Reply by znguvnf** [reply]
Any update to this issue? Having the same problem, Akka.Analyzers preventing Akka.NET to be installed on .NET Framework.

**Reply by Aaronontheweb** [reply, marked as answer]
Ah sorry, but yes - we had another version of this same discussion pop up as a bona fide GitHub issue: https://github.com/akkadotnet/akka.net/issues/7307

TL;DR; this is an issue with the version of MSBuild you're using, not the version of .NET. If you upgrade to an MSBuild15-style project (which you can do without changing your .NET version) this issue should go away. The alternative is for us to downgrade Akka.Analysers to an older version of Roslyn.

xunit ran into this same issue and the .NET team is still discussing making some fixes to the runtime for older-style projects - you can follow those links on my comment here: https://github.com/akkadotnet/akka.net/issues/7307#issuecomment-2253195327
